pkg/mic/mic.go

```diff
@@ -185,7 +186,7 @@ func (c *Client) Start(exit <-chan struct{}) {
 
 	wg.Add(1)
 	go func() {
-		c.CRDClient.Start(exit)
+		c.CRDClient.Start(exit, Log{})
```

Comment:
I was thinking we should embed a struct in miclient and use that here and other places..

Reply:
ok, I'll take a look.